.github/workflows/CI.yml

```
@@ -10,6 +10,7 @@ jobs:
   test:
     name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
     runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.version == 'nightly' }}
```

Did it work? I am not sure...
```suggestion
    continue-on-error: ${{ matrix.julia-version == 'nightly' }}
```
Not sure, just saw this elsewhere.

---

I think in the `matrix.XX` the `XX` should match the name of the variable as defined above, in this case `version` see [here](https://github.com/JuliaIntervals/IntervalLinearAlgebra.jl/blob/main/.github/workflows/CI.yml#L11) . 

My understanding is that if it "works" then you will still see individual failures in the summary in the PR but when you merge the readme badge will be green if failure is only on nightly

---

Okay, then better undo it.